ci/travis-27.yaml

```diff
@@ -0,0 +1,49 @@
+name: pandas
+channels:
```

Comment:
can we have one of the main builds NOT use conda-forge (maybe the 3.5 one)? in the original we had *only* the 3.6 build using it.